03-vectors/vectors-practice-1.Rmd

```diff
@@ -42,6 +53,13 @@ opts_chunk$set(warning=FALSE,message=FALSE)
 \{1, 1, 2, 2, 3, 3, 1, 1, 2, 2, 3, 3, 1, 1, 2, 2, 3, 3\}
 \]
 
+```{r, eval=TRUE}
+rep(1:3,3)
+rep(1:3, each=2)
+rep(1:3, c(2,1,2))
+rep(1:3, len=7)
+rep(1:3, each=2, times=3)
```

Comment:
I had no idea you could use the each argument here. I did nested rep commends for this last one, but this code is much prettier!